Q_AND_A.md

```diff
@@ -0,0 +1,24 @@
+
+# DWN Q&A
```

Comment:
Nit: What do you think of `FAQ` instead?

Reply:
i didn’t do it because I didn’t think these are “frequently” asked questions…. LOL